### PR TITLE
feat: Implemented Torch front-end empty_strided function

### LINKED ISSUE #22935: empty_strided

### PR DESCRIPTION
**Pull request by rohitkg83**
# PR Description

#22935 - Implemented Torch Frontend function empty_strided and added unit test for it.

Close #22935 

## Checklist

- [x] Did you add a function? - Yes
- [x] Did you add the tests? - Yes
- [x] Did you follow the steps we provided? - Yes

### Socials:

https://twitter.com/rkg830926

### CODE REVIEW

File: ivy/functional/frontends/torch/creation_ops.py [hunk 1]
```diff
@@ -93,6 +93,55 @@ def empty_like(
...
+    size,
+    stride,
+    *,
+    dtype=None,
+    layout=None,
+    device=None,
+    requires_grad=False,
+    pin_memory=False,
+):
+    if dtype == "int8":
```
Comment (bipinKrishnan) [inline ivy/functional/frontends/torch/creation_ops.py:107]: If these are unsupported dtypes, you could make use of the [ivy decorator](https://github.com/asuzukosi/ivy/blob/d92979a49735bf2aa3741b28df8157fc69e1ce55/ivy/func_wrapper.py#L1632C9-L1632C9) for this use-case.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py [hunk 2]
```diff
@@ -323,6 +323,39 @@ def test_torch_empty_like(
...
+            backend_to_test=backend_fw,
+            frontend=frontend,
+            test_flags=test_flags,
+            fn_tree=fn_tree,
+            on_device=on_device,
+            size=size,
+            stride=stride,
+            test_values=False,
+        )
+    except Exception as e:
```
Comment (bipinKrishnan) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py:352]: The exception catching is not expected here, we won't be able to catch the failures correctly. So you could remove the try-except statement. May be you could refer to other test implementations in this file.

File: ivy/functional/frontends/torch/creation_ops.py (hunk 1 above)
Comment (rohitkg83) [reply ivy/functional/frontends/torch/creation_ops.py]: Thanks for reviewing. I have removed the if elif statements as empty_strided functions supports all dtypes. Can you please review again?

File: ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py (hunk 2 above)
Comment (rohitkg83) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py]: Thanks again, my bad. I have removed try-except. Can you please review again?

**Comment by bipinKrishnan** [comment]
LGTM! Thanks for the contribution @rohitkg83 🚀 .

**Pull request merged**